ansible/playbooks/windows.yml

```diff
@@ -326,6 +373,16 @@
       when: (java8_installed.stat.exists == false)
       tags: java8
 
+    - name: Add JAVA_HOME/bin to Path
+      win_path:
+        name: Path
+        elements: 
+          - 'C:\Program Files\Java\jdk8u144-b01\bin'
+        scope: machine
+        state: present
+      when: (java8_installed.stat.exists == false)
+      tags: java8
+
```

Comment:
In what situation would java8 not exist?

Reply:
New machine